skimage/transform/_warps.py

```diff
@@ -657,8 +657,38 @@ def _clip_warp_output(input_image, output_image, order, mode, cval, clip):
             output_image[cval_mask] = cval
 
 
+def warp_fast(image, H, output_shape=None,
+              order=1, mode='constant', cval=0, dtype=np.float64):
+    """Projective transformation (homography).
+
+    This is a Python wrapper for the Cython function ``_warp_fast``.
+    It prepares the data and allocates the necessary memory.
+
+    Parameters
+    ----------
+    output_shape : tuple (rows, cols), optional
+        Shape of the output image generated (default None).
+    dtype :
+        The numric type of the output image.
```

Comment:
This docstring is pretty woefully inaccurate...

Reply:
What needs to be added?